Makefile

```diff
@@ -85,38 +85,24 @@ $(FC_TEST_DATA_PATH)/ltag:
 	GO111MODULE=off GOBIN=$(abspath $(FC_TEST_DATA_PATH)) \
 	go get github.com/kunalkushwaha/ltag
 
-tools/firecracker-builder-stamp: tools/docker/Dockerfile
-	docker build \
-		-t localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG) \
-		-f tools/docker/Dockerfile \
-		tools/docker
-	touch $@
+$(FIRECRACKER_DIR):
+	- git clone https://github.com/firecracker-microvm/firecracker.git $(FIRECRACKER_DIR)
 
 .PHONY: test-images
 test-images: $(FIRECRACKER_BIN) $(JAILER_BIN)
 
-$(FIRECRACKER_BIN) $(JAILER_BIN): tools/firecracker-builder-stamp
-	mkdir -p build
-	docker run --rm -it \
-		--user $(UID):$(GID) \
-		--volume $(CURDIR)/build:/artifacts \
-		--volume $(CARGO_CACHE_VOLUME_NAME):/usr/local/cargo/registry \
-		-e HOME=/tmp \
-		--workdir=/firecracker \
-		localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG) \
-		cargo build --release \
-		--target-dir=/artifacts --target $(FIRECRACKER_TARGET) \
-		-p firecracker -p jailer
-	cp build/$(FIRECRACKER_TARGET)/release/firecracker $(FIRECRACKER_BIN)
-	cp build/$(FIRECRACKER_TARGET)/release/jailer $(JAILER_BIN)
+$(FIRECRACKER_BIN) $(JAILER_BIN): $(FIRECRACKER_DIR)
+	# Strip currently commented out because it's being fixed in Firecracker
+	# and is under review.
+	$(FIRECRACKER_DIR)/tools/devtool -y build --release #&& \
+		#$(FIRECRACKER_DIR)/tools/devtool strip
+	cp $(FIRECRACKER_DIR)/build/cargo_target/$(FIRECRACKER_TARGET)/release/firecracker $(FIRECRACKER_BIN)
+	cp $(FIRECRACKER_DIR)/build/cargo_target/$(FIRECRACKER_TARGET)/release/jailer $(JAILER_BIN)
```

Comment:
How about having `install` target on the devtool script? Right now the script is hiding Cargo only partially.

Reply:
How would you see the `install` command working for multiple binaries?

Reply:
Installing both of them under `/usr/local/bin`? The location should be customizable though.

Reply:
That would work. Something like `./devtool install --path /path/to/custom/dir` is doable.

We will also need to add the `seccompiler` binary to the install process.

Reply:
@kzys Why do you prefer `install` over `cp`? Also could we merge this update first and see if it get the pipeline fixed or maybe wait for a little bit?

Reply:
I'm fine merging the change as is and will do.

Using the devtool is good because it abstracts away Firecracker's build process. We don't have to know Cargo, Rust and other stuff, so Compute Capsule team can do whatever they want, without worrying about breaking downstream dependencies.

However if we hardcode the location of the binaries, [the abstraction is leaky](https://www.joelonsoftware.com/2002/11/11/the-law-of-leaky-abstractions/) than what it could be.

Reply:
Thanks @kzys. Always learn from you. Like this sentence from the article, "So the abstractions save us time working, but they don't save us time learning."

Reply:
Added a PR in Firecracker to support the `install` subcommand [here](https://github.com/firecracker-microvm/firecracker/pull/2575).